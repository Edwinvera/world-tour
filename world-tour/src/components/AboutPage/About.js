import React from 'react'
import styled from 'styled-components'


const Main = styled.main`
  display: flex;
  justify-content: center;
  text-align: center;
  margin: 225px 0 100px 0;
`
const Section = styled.section`
  width: 900px;
  border-radius: 360px; 
  background-color: rgb(183,176,159);
`
const AboutSection = styled.section`
  display: flex;
  flex-direction: column;
  align-items: center;
  padding: 0px 70px;
`
const DataSection = styled(AboutSection)`
  margin: 15px 0 30px 0
`
const Header = styled.h1`
  font-family: 'Manrope', sans-serif;
  font-size: 32px;
  letter-spacing: 2px;
  margin: 25px 0 0 0;
  color: rgb(16, 23, 130)
`
const Paragraph = styled.p`
  font-family: 'Roboto', sans-serif;
  font-size: 21px;
  text-align: left;
  margin: 15px 0;
  letter-spacing: 3px;
  line-height: 1.6;
  color: rgb(60,95,60)
`
const Link = styled.a`
  color: black;
  text-decoration: none
`

function About() {
  return (
    <div>
      <Main>
        <Section>
          <AboutSection>
            <Header>About</Header>
            <Paragraph>"World Tour" allows users to take a satellite journey to both familiar and foreign lands of Earth. You can randomly teleport or get specific in your search for regional data.</Paragraph>
          </AboutSection>

          <DataSection>
            <Header>Data & Tech</Header>
            <Paragraph>This application is written in React.js and pulls data from the RESTful API at <Link href="https://restcountries.eu/">https://restcountries.eu/</Link></Paragraph>
          </DataSection>
        </Section>
      </Main>
    </div>
  )
}

export default About
