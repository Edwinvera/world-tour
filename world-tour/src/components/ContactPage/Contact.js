import React from 'react'
import styled from 'styled-components'

const Main = styled.main`
  display: flex;
  justify-content: center;
  text-align: center;
  margin: 225px 0;
`
const Section = styled.section`
  width: 900px;
  border-radius: 360px; 
  background-color: rgb(183,176,159);
  padding: 0 0 30px 0
`
const Header = styled.h1`
  font-family: 'Manrope', sans-serif;
  font-size: 32px;
  letter-spacing: 2px;
  margin: 25px 0 0 0;
  color: rgb(16, 23, 130)

`
const Paragraph = styled.p`
  font-family: 'Roboto', sans-serif;
  font-size: 24px;
  margin: 25px 0;
  letter-spacing: 3px;
  color: black;
`
const Link = styled.a`
  color: black;
  text-decoration: none
`

function Contact() {
  return (
    <div>
      <Main>
        <Section>
          <Header>Contact</Header>
          <Paragraph>Developer | Stefon Simmons</Paragraph>
          <Paragraph>Email | <Link href="mailto:stefonsimmons1@gmail.com">stefonsimmons1@gmail.com</Link></Paragraph>
          <Paragraph>GitHub | <Link href="http://github.com/StefonSimmons/">github.com/StefonSimmons</Link></Paragraph>
        </Section>
      </Main>
    </div>
  )
}

export default Contact
