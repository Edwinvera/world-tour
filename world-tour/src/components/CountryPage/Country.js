import React from 'react'
import {useParams, Link} from 'react-router-dom'
import CountUp from 'react-countup'
import styled from 'styled-components'

const Main = styled.main`
  display: flex;
  align-items: center;
  flex-direction: column;
  text-align: left;
  margin: 250px 0 0 0;
  padding-bottom: 80px
`
const Summary = styled.section`
  width: 500px;
  background-color: rgba(20,20,20,.1);
  box-shadow: 5px 10px 20px rgb(20,20,20);
  padding: 40px;
  margin-bottom: 50px
`
const Header = styled.h1`
  margin-bottom: 5px;
  font-family: 'Nunito Sans', sans-serif;
  font-size: 36px;
  letter-spacing: 5px;
`
const NativeName = styled.i`
  font-size: 12px;
  letter-spacing: 5px;
`
const Divider = styled.hr`
  width: 500px;
  margin: 18px 0px
`
const Table = styled.table`
  width: 500px
`
const Label = styled.td`
  font-family: 'Nunito Sans', sans-serif;
  font-size: 18px;
  line-height: 2
`
const Details = styled.td`
  font-family: sans-serif;
  font-size: 18px;
  line-height: 2;
  text-align: right
`
const Flag = styled.img`
  width: 600px;
  border: grey solid 2px
`
const ReturnBtn = styled.button`
  font-size: 18px;
  border: solid 1px rgba(20,20,20,.2);
  border-radius: 15px;
  background-color: rgba(60,60,155,.1);
  cursor: pointer;
`
const RegionLink = styled(Link)`
`

function Country({regions}) {

  
  const {countryName} = useParams()
  let countryData = regions.filter(r => r.name === countryName)
  let country = countryData[0]
  console.log(country)


  return (
    <div>
      <Main>
        <Summary>
          <RegionLink to={`/region/${country.region}`}>
            <ReturnBtn>{`${`â¬…`} ${country.region}`}</ReturnBtn>
          </RegionLink>
          <Header>{countryName}</Header>
          <NativeName>( {country.nativeName} )</NativeName>
          <Divider/>
          <Table>
            <tbody>
              <tr>
                <Label>{`Population in ${countryName}:`}</Label>
                <Details>
                  <CountUp
                    end={country.population}
                    separator=','
                  />
                </Details>
              </tr>
              <tr>
                <Label>Area of Land (mi<sup>2</sup>):</Label>
                <Details>
                  <CountUp
                    end={country.area}
                    separator=','
                  />
                </Details>
              </tr>
              <tr>
                <Label>Capital:</Label>
                <Details>{country.capital}</Details>
              </tr>
              <tr>
                <Label>Currency (Symbol):</Label>
                <Details>{country.currencies[0].code} ({country.currencies[0].symbol})</Details>
              </tr>
              <tr>
                <Label>Region:</Label>
                <Details>{country.region}</Details>
              </tr>
              <tr>
                <Label>Sub Region:</Label>
                <Details>{country.subregion}</Details>
              </tr>
            </tbody>
          </Table>
          <Divider />
        </Summary>
        <a
          href={`https://www.google.com/maps/place/${countryName}/@${country.latlng[0]},${country.latlng[1]},6z/`}
          target="_blank"
          rel="noopener noreferrer"
        >
          <Flag src={country.flag} alt={countryName}/>
        </a>  
      </Main>
    </div>
  )
}

export default Country
