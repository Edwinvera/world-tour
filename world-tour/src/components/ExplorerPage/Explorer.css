.Africa{
  background-image: url("../../images/ExplorerRegions/Africa.jpg");
  background-repeat: no-repeat;
  background-size: cover;
}

.Europe{
  background-image: url("../../images/ExplorerRegions/Europe.jpg");
  background-repeat: no-repeat;
  background-size: cover;
}

.Americas{
  background-image: url("../../images/ExplorerRegions/Americas.jpg");
  background-repeat: no-repeat;
  background-size: cover;
}

.Asia{
  background-image: url("../../images/ExplorerRegions/Asia.jpg");
  background-repeat: no-repeat;
  background-size: cover;
}
.Oceania{
  background-image: url("../../images/ExplorerRegions/Oceania.jpg");
  background-repeat: no-repeat;
  background-size: cover; 
}
.Polar{
  background-image: url("../../images/ExplorerRegions/Polar.jpg");
  background-repeat: no-repeat;
  background-size: cover;
}
