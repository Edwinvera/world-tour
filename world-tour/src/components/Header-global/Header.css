/* @import url('') */
/* Earth logo ⬇ */

.earth-logo {
  position: absolute;
  height: 30vmin;
  top: 20px;
  right: 42vw;
  cursor: pointer;
  animation: earth-logo-spin infinite 20s linear;
}

@keyframes earth-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}

/* Earth logo ⬆ */

.app-header {
  display: flex;
  justify-content: flex-end;
  background-color: #282c34;
  position: relative;
}
li{
  display: inline-block;
  list-style-type: none;
  padding: 15px;
}
ul li:nth-child(n+3){
  border-left: solid 1px grey
}
.menu-item{
  display: inline block;
  color: white;
  text-decoration: none;
  text-align: center;
  margin: 25px
}

nav{
  height: 80px;
}

#hamburger{
  display: none;
}

@media(max-width: 861px){
  .menu-item-container{
    display: none; 
    background-color: #282c34;
  }
  .menu-item{
    margin: 25px 15px;
  }
  #hamburger{
    display: inline-block;
    width: 40px;
    padding: 6px 40px
  }
}

