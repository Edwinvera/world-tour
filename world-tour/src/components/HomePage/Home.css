main{
  display: flex;
  justify-content: center;
  /* align-content: center; */
  text-align: center;
  bottom: 50px
}