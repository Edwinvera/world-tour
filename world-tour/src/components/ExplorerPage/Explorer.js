import React, {useState} from 'react'
import { Link } from 'react-router-dom'
import './Explorer.css'
import styled from 'styled-components'

const Main = styled.main`
  justify-content: center;
  text-align: center;
  margin: 225px 0
`

const SectionRandom = styled.section`
  display: flex;
  flex-direction: column;
  align-items: center;
`

const RandoBtn = styled.button`
  margin: 15px 0px;
  height: 35px;
  background-color: rgb(17,124,158);
  color: white;
  border-radius: 5px;
  font-size: 18px;
  cursor: pointer
`
const RandoBox = styled.div`
  display: flex;
  align-items: center;
  justify-content: center;
  height: 50px;
  width: 300px;
  border: blue dotted 2px;
  background-color: rgba(17,124,158,.2)
`
const RandoCountry = styled.h5`
  font-family: 'Montserrat', sans-serif;
  font-size: 18px
`
const CountryLink = styled(Link)`
  text-decoration: none;
  color: black
`
const RegionLink = styled(CountryLink)`
  text-decoration: none;
  color: black
`
const SectionRegion = styled(SectionRandom)`
  margin: 50px 0
`
const RegionContainer = styled.div`
  display: flex;
  align-items: center;
  justify-content: center;
  margin: 10px;
  height: 300px;
  width: 1000px;
`
const Region = styled.h2`
  font-family: 'Nunito Sans', sans-serif;
  font-size: 36px;
  letter-spacing: 5px;
  padding: 0px 5px;
  color: white;
  background-color: rgba(20,20,20,.1);
  box-shadow: 5px 10px 20px rgb(20,20,20);
`


function Explorer({ uniqueRegions, countries }) {
  
  const [randoCountry, updateCountry] = useState('--?--')

  function handleRandomBtn(e) {
    const countryCount = countries.length
    const randomNumber = Math.floor(Math.random() * countryCount)
    const randomCountry = countries.filter((country,i) => countries.indexOf(country) === randomNumber)

    updateCountry(randomCountry)
    // console.log(randomCountry)
  }

  return (
    <div>
      <Main>
        <SectionRandom>

          <RandoBtn onClick= {(e) => handleRandomBtn()}>Visit a Random Country</RandoBtn>
          
          <RandoBox>
            <CountryLink to='/country/:alpha3code'>
              <RandoCountry>{randoCountry}</RandoCountry>
            </CountryLink>
          </RandoBox>
          
        </SectionRandom>
        <SectionRegion>
          {/* A map loop for regions here. Include a className so I can style the region names with a background image*/}
          {uniqueRegions.map((region, id) => 
            <RegionContainer className={region}>
              <RegionLink to='/region/:alphacode3' key={id}>
                <Region>{region}</Region>
              </RegionLink>
            </RegionContainer>
          )} 
        </SectionRegion>  
      </Main>
      
      
    </div>
  )
}

export default Explorer
