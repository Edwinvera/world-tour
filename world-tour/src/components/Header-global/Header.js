import React, {useState} from 'react'
import {Link} from 'react-router-dom'
import earth from '../../images/Emblem-earth.svg'
import hamburger from '../../images/hamburger.png'
import './Header.css'
// import styled from 'styled-components'



function Header() {

  const [open, updateOpen] = useState(false) 


  return (
    <div>

      <header className="app-header">
        <nav>
          <ul>
            {/* <li id= "hamburger-container"> DONT NEED*/}
            <img id="hamburger" src={hamburger} alt="hamburger menu" onClick={(e) => updateOpen(!open)}/>
            {/* {console.log(open)} */}
            {/* </li> DONT NEED*/}
            <li className= "menu-item-container" style={open ? { display:"block" }: {display:""}}>
              <Link to='/' className="menu-item" style={open ? { display:"block" }: {display:""}}>
                  Home
              </Link>
            </li>
            <li className= "menu-item-container" style={open ? { display:"block" }: {display:""}}>
              <Link to='/about' className="menu-item" style={open ? { display:"block" }: {display:""}}>
                  About
              </Link>
            </li>
            <li className= "menu-item-container" style={open ? { display:"block" }: {display:""}}>
              <Link to='/contact' className="menu-item" style={open ? { display:"block" }: {display:""}}> 
                  Contact
              </Link>
            </li>
          </ul>
        </nav>
        <div>
          <Link to='/explorer'>
            <img src={earth} className="earth-logo" alt="logo" />
          </Link>
        </div>
        
      </header>
    </div>
  )
}

export default Header
