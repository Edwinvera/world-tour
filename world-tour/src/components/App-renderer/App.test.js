import Header from '../Header-global/Header'
import Home from '../HomePage/Home'
import About from '../AboutPage/About'
import Contact from '../ContactPage/Contact'
import Explorer from '../ExplorerPage/Explorer'
import Region from '../RegionPage/Region'
import Country from '../CountryPage/Country'
import Footer from '../Footer-global/Footer'

// import renderer from 'react-test-renderer'
// import 'jest-styled-components'

// test('Home component renders', () => {
//   const HomeComponent = renderer.create(<Home/>).toJSON()
//   expect(HomeComponent).toMatchSnapshot()
// })


// test('Explorer component renders', () => {
//   const ExplorerComponent = renderer.create(<Explorer/>).toJSON()
//   expect(ExplorerComponent).toMatchSnapshot()
// })


// test('About component renders', () => {
//   const AboutComponent = renderer.create(<About/>).toJSON()
//   expect(AboutComponent).toMatchSnapshot()
// })


