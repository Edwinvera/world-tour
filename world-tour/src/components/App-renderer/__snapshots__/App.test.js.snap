// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component renders 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  margin: 250px 0 100px 0;
}

.c1 {
  width: 900px;
  border-radius: 360px;
  background-color: rgb(183,176,159);
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 0px 70px;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 0px 70px;
  margin: 15px 0 30px 0;
}

.c3 {
  font-family: 'Manrope',sans-serif;
  font-size: 32px;
  -webkit-letter-spacing: 2px;
  -moz-letter-spacing: 2px;
  -ms-letter-spacing: 2px;
  letter-spacing: 2px;
  margin: 25px 0 0 0;
  color: rgb(16,23,130);
}

.c4 {
  font-family: 'Roboto',sans-serif;
  font-size: 21px;
  text-align: left;
  margin: 15px 0;
  -webkit-letter-spacing: 3px;
  -moz-letter-spacing: 3px;
  -ms-letter-spacing: 3px;
  letter-spacing: 3px;
  line-height: 1.6;
  color: rgb(60,95,60);
}

.c6 {
  color: black;
  -webkit-text-decoration: none;
  text-decoration: none;
}

@media (max-width:861px) {
  .c1 {
    width: 600px;
    margin-bottom: 45px;
  }
}

@media (max-width:867px) {
  .c3 {
    font-size: 28px;
  }
}

@media (max-width:365px) {
  .c3 {
    font-size: 24px;
  }
}

@media (max-width:867px) {
  .c4 {
    font-size: 15px;
    -webkit-letter-spacing: 1px;
    -moz-letter-spacing: 1px;
    -ms-letter-spacing: 1px;
    letter-spacing: 1px;
  }
}

@media (max-width:365px) {
  .c4 {
    font-size: 15px;
  }
}

<div>
  <main
    className="c0"
  >
    <section
      className="c1"
    >
      <section
        className="c2"
      >
        <h1
          className="c3"
        >
          About
        </h1>
        <p
          className="c4"
        >
          "World Tour" allows users to take a satellite journey to both familiar and foreign lands of Earth. You can randomly teleport or get specific in your search for regional data.
        </p>
      </section>
      <section
        className="c5"
      >
        <h1
          className="c3"
        >
          Data & Tech
        </h1>
        <p
          className="c4"
        >
          This application is written in React.js and pulls data from the RESTful API at 
          <a
            className="c6"
            href="https://restcountries.eu/"
          >
            https://restcountries.eu/
          </a>
        </p>
      </section>
    </section>
  </main>
</div>
`;

exports[`Contact component renders 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  margin: 225px 0;
}

.c1 {
  width: 900px;
  border-radius: 360px;
  background-color: rgb(183,176,159);
  padding: 30px;
}

.c2 {
  font-family: 'Manrope',sans-serif;
  font-size: 32px;
  -webkit-letter-spacing: 2px;
  -moz-letter-spacing: 2px;
  -ms-letter-spacing: 2px;
  letter-spacing: 2px;
  margin: 25px 0 0 0;
  color: rgb(16,23,130);
}

.c3 {
  font-family: 'Roboto',sans-serif;
  font-size: 24px;
  margin: 25px 0;
  -webkit-letter-spacing: 3px;
  -moz-letter-spacing: 3px;
  -ms-letter-spacing: 3px;
  letter-spacing: 3px;
  color: black;
}

.c4 {
  color: black;
  -webkit-text-decoration: none;
  text-decoration: none;
}

@media (max-width:365px) {
  .c2 {
    font-size: 28px;
  }
}

@media (max-width:365px) {
  .c2 {
    font-size: 24px;
    margin: 10px 0 0 0;
  }
}

@media (max-width:865px) {
  .c3 {
    font-size: 15px;
  }
}

@media (max-width:365px) {
  .c3 {
    font-size: 12px;
  }
}

<div>
  <main
    className="c0"
  >
    <section
      className="c1"
    >
      <h1
        className="c2"
      >
        Contact
      </h1>
      <p
        className="c3"
      >
        Developer | Stefon Simmons
      </p>
      <p
        className="c3"
      >
        Email | 
        <a
          className="c4"
          href="mailto:stefonsimmons1@gmail.com"
        >
          stefonsimmons1@gmail.com
        </a>
      </p>
      <p
        className="c3"
      >
        GitHub | 
        <a
          className="c4"
          href="http://github.com/StefonSimmons/"
        >
          github.com/StefonSimmons
        </a>
      </p>
    </section>
  </main>
</div>
`;

exports[`Footer component renders 1`] = `
.c0 {
  position: absolute;
  bottom: 0px;
  width: 100%;
  height: .1vh;
  text-align: center;
  opacity: .9;
}

.c1 {
  background-color: rgb(28,28,28);
  padding: 1px 0;
}

.c2 {
  font-family: 'Merriweather Sans',sans-serif;
  font-size: 16px;
  color: #DAE2DF;
}

<div>
  <footer
    className="c0"
  >
    <div
      className="c1"
    >
      <h6
        className="c2"
      >
        Â© Stefon Simmons
      </h6>
    </div>
  </footer>
</div>
`;

exports[`Home component renders 1`] = `
.c0 {
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  margin: 250px 0 0 0;
  padding-bottom: 5rem;
}

.c2 {
  color: #403F4C;
}

.c1 {
  font-family: 'Montserrat',sans-serif;
  font-size: 45px;
  color: #282c34;
}

.c5 {
  margin-bottom: 0px;
  font-family: 'Manrope',sans-serif;
  font-size: 30px;
  -webkit-letter-spacing: 10px;
  -moz-letter-spacing: 10px;
  -ms-letter-spacing: 10px;
  letter-spacing: 10px;
  color: rgb(16,23,100);
}

.c6 {
  margin: 5px 0 50px 0;
  font-family: 'Montserrat',sans-serif;
  font-size: 24px;
  -webkit-letter-spacing: 20px;
  -moz-letter-spacing: 20px;
  -ms-letter-spacing: 20px;
  letter-spacing: 20px;
}

.c3 {
  font-family: 'Source Sans Pro',sans-serif;
  font-size: 90px;
  color: #282c34;
}

.c4 {
  width: 500px;
  margin-bottom: 80px;
  height: 1px;
  background-color: #1B2432;
}

@media (max-width:861px) {
  .c1 {
    font-size: 36px;
  }
}

@media (max-width:659px) {
  .c1 {
    font-size: 24px;
  }
}

@media (max-width:365px) {
  .c1 {
    font-size: 18px;
  }
}

@media (max-width:861px) {
  .c5 {
    font-size: 24px;
    -webkit-letter-spacing: 7px;
    -moz-letter-spacing: 7px;
    -ms-letter-spacing: 7px;
    letter-spacing: 7px;
  }
}

@media (max-width:659px) {
  .c5 {
    font-size: 18px;
  }
}

@media (max-width:365px) {
  .c5 {
    font-size: 15px;
    -webkit-letter-spacing: 7px;
    -moz-letter-spacing: 7px;
    -ms-letter-spacing: 7px;
    letter-spacing: 7px;
  }
}

@media (max-width:659px) {
  .c6 {
    font-size: 18px;
  }
}

@media (max-width:365px) {
  .c6 {
    font-size: 15px;
    -webkit-letter-spacing: 15px;
    -moz-letter-spacing: 15px;
    -ms-letter-spacing: 15px;
    letter-spacing: 15px;
  }
}

@media (max-width:659px) {
  .c3 {
    font-size: 24px;
  }
}

@media (max-width:365px) {
  .c3 {
    font-size: 18px;
  }
}

@media (max-width:659px) {
  .c4 {
    width: 400px;
    margin-bottom: 60px;
  }
}

@media (max-width:365px) {
  .c4 {
    width: 250px;
    margin-bottom: 60px;
  }
}

<main
  className="c0"
>
  <section>
    <h1
      className="c1"
    >
      <span
        className="c2"
      >
        Welcome Home
      </span>
       
      <span
        className="c3"
      >
        |
      </span>
       3rd-Star Explorer
    </h1>
    <hr
      className="c4"
    />
    <h2
      className="c5"
    >
      Population on Earth
    </h2>
    <p
      className="c6"
    >
      <span />
    </p>
    <h2
      className="c5"
    >
      Area of Land (mi
      <sup>
        2
      </sup>
      )
    </h2>
    <p
      className="c6"
    >
      <span />
    </p>
    <h2
      className="c5"
    >
      Recorded Regions
    </h2>
    <p
      className="c6"
    >
      <span />
    </p>
  </section>
</main>
`;

exports[`Region component renders 1`] = `
.c0 {
  font-family: 'Nunito Sans',sans-serif;
  font-size: 36px;
  -webkit-letter-spacing: 5px;
  -moz-letter-spacing: 5px;
  -ms-letter-spacing: 5px;
  letter-spacing: 5px;
  padding: 0px 5px;
  color: white;
  background-color: rgba(20,20,20,.1);
  box-shadow: 5px 10px 20px rgb(20,20,20);
}

<h2
  className="c0"
/>
`;
